frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/usePodContainerLogState.ts

```diff
@@ -19,20 +19,27 @@ const usePodContainerLogState = (
   podContainers: PodContainer[];
   podContainerStatuses: PodContainerStatuses;
   selectedContainer: PodContainer | null;
+  defaultContainerName: string;
   setSelectedContainer: (podContainer: PodContainer | null) => void;
 } => {
   const { namespace } = usePipelinesAPI();
   const [pod, podLoaded, podError] = usePod(namespace, podName);
   const { containers: podContainers, containerStatuses: podContainerStatuses } =
     getPodContainers(pod);
   const [selectedContainer, setSelectedContainer] = React.useState<PodContainer | null>(null);
+  const defaultContainerName =
+    pod?.metadata.annotations?.['kubectl.kubernetes.io/default-container'] ?? '';
```

Comment:
what happens if this annotation is not defined. It looks like we would just render an empty string.

Reply:
@Gkrumbach07 Thanks, I have updated the dropdown to hide the task group when annotation is undefined